applications/luci-app-vnstat2/luasrc/view/vnstat2/graphs.htm

```diff
@@ -0,0 +1,55 @@
+<%#
+ This is free software, licensed under the Apache License, Version 2.0
+-%>
+
+<%-
+
+local util = require "luci.util"
+local json = require "luci.jsonc"
+
+
+local ifaces = {}
+
+local data = util.exec("vnstat --json f 1 2>/dev/null")
+if #data > 0 and data:sub(1,1) == "{" then
```

Comment:
I think this is not necessary you could check the return value of json.parse directly.
https://github.com/openwrt/luci/blob/33597c79ac5d7ee917f8452cc9c26fd546c2cf1f/applications/luci-app-attendedsysupgrade/luasrc/view/attendedsysupgrade.htm#L12-L15